docs/src/components/document-renderer.tsx

```diff
@@ -108,6 +108,20 @@ const getRenderers = (
     divider: () => {
       return <hr className="border-keystatic-gray my-2" />;
     },
+    layout: ({ layout, children }) => (
+      <div
+        className="grid gap-4"
+        style={{
+          gridTemplateColumns: layout.map(x => `${x}fr`).join(' '),
```

Comment:
Do you mind if we avoid single letter variables? I find them hard to read 😅

Reply:
Fair call, I did a cheeky copy/paste from the default document-render layout component in core for this one to add our tailwind classes 🙈

What name would make sense? `colSpan` is the best I can come up with but I'm not sure that is actually correct 🤔

Reply:
`layoutColumns`, or `columns` perhaps, not exactly what it is but close. It is like a column index with possible span information 😅

Reply:
I would go `columnSize` or `columnFraction`.

Reply:
columnFractions? 😅

Reply:
layoutFraction in singular would make sense to me actually, but doesn't really matter :D

Reply:
The people have spoken